cmd/kubeadm/app/cmd/defaults.go

```diff
@@ -29,8 +29,7 @@ import (
 )
 
 var (
-	// Maximum version when using AllowAll as the default authz mode. Everything above this will use RBAC by default.
-	allowAllMaxVersion = semver.MustParse("1.6.0-alpha.0")
+	minK8sVersion = semver.MustParse("1.6.0-alpha.1")
```

Comment:
It can't be a `const` since it's assigned by an external package `semver`.

And it's not the same as the fallback version either; in the future we might want to set the fallback version to be `v1.6.2` for instance, but the minimum required version would still be `v1.6.0` or the like